### PR TITLE
Added apply_slice function to enable multiple versions of Slice

### PR DESCRIPTION
**Pull request by prabhat00155**

### CODE REVIEW

File: onnxutils/onnxconverter_common/onnx_ops.py [hunk 1]
```diff
@@ -497,6 +497,35 @@ def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=N
         # output = a * d
         apply_mul(scope, [aName, dName], output_name, container)
 
+def apply_slice(scope, input_name, output_names, container, starts, ends,
```
Comment (vinitra) [inline onnxutils/onnxconverter_common/onnx_ops.py]: Since slice only has one output, call it output_name for consistency.

Comment (prabhat00155) [reply onnxutils/onnxconverter_common/onnx_ops.py]: Good point, fixed.

File: onnxutils/onnxconverter_common/onnx_ops.py [hunk 2]
```diff
@@ -497,6 +497,37 @@ def apply_scaled_tanh(scope, input_name, output_name, container, operator_name=N
...
+        inputs.append(starts_name)
+        inputs.append(ends_name)
+        if axes:
+            axes_name = scope.get_unique_variable_name('axes')
+            container.add_initializer(axes_name, onnx_proto.TensorProto.INT64,
+                                      [len(axes)], axes)
+            inputs.append(axes_name)
+        if steps:
+            if not axes:
+                inputs.append('')
```
Comment (vinitra) [inline onnxutils/onnxconverter_common/onnx_ops.py]: In this case do you still need to add an initializer for axes?

Comment (prabhat00155) [reply onnxutils/onnxconverter_common/onnx_ops.py]: No, we are passing empty string as a placeholder for axes so that we can pass steps.

**Pull request merged**